### ISSUE TITLE
[BUG] GraphQL API Error

### ISSUE THREAD

**Issue by pigOnTuyere** [voxel51/fiftyone#3570]
### System information

-   **OS Platform and Distribution** (e.g., Linux Ubuntu 16.04):win11
-   **Python version** (`python --version`):3.7.16
-   **FiftyOne version** (`fiftyone --version`):0.22.0
-   **FiftyOne installed from** (pip or source):pip install fiftyone

### Describe the problem
When I use the following code,there is no Error message in the console interface, but the fiftyone webboard interface is blank and does not display anything, and the desktop displays GraphQL API Error
```python
import fiftyone as fo
import fiftyone.zoo as foz

dataset = foz.load_zoo_dataset("quickstart")
session = fo.launch_app(dataset,desktop=True)
#session = fo.launch_app(dataset)
session.wait()
```
![image](https://github.com/voxel51/fiftyone/assets/145239920/ba2d4294-fd97-4b29-9c5c-6a0b2fa705a8)

![image](https://github.com/voxel51/fiftyone/assets/145239920/30f61524-d41b-4d65-a19e-31f6905632f2)

![image](https://github.com/voxel51/fiftyone/assets/145239920/6f21aa47-a06f-4aa1-b65f-ce9f6aa83399)

### Other info/logs

```commandline
File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\graphql\execution\execute.py", line 1036, in await_result
    return build_response(await result, errors)  # type: ignore
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\graphql\execution\execute.py", line 451, in get_results
    await gather(*(results[field] for field in awaitable_fields)),
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\graphql\execution\execute.py", line 535, in await_result
    self.handle_field_error(error, return_type)
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\graphql\execution\execute.py", line 569, in handle_field_error
    raise error
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\graphql\execution\execute.py", line 528, in await_result
    return_type, field_nodes, info, path, await result
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\strawberry\schema\schema_converter.py", line 513, in _async_resolver
    return await await_maybe(_get_result(_source, strawberry_info, **kwargs))
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\strawberry\schema\schema_converter.py", line 496, in _get_result
    source=_source, info=info, kwargs=kwargs
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\strawberry\schema\schema_converter.py", line 436, in _get_arguments
    config=self.config,
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\strawberry\arguments.py", line 215, in convert_arguments
    scalar_registry=scalar_registry,
  File "D:\Users\jack\anaconda3\envs\fo\lib\site-packages\strawberry\arguments.py", line 182, in convert_argument
    return type_(**kwargs)
```

```commandline
GraphQL API Error
    at GHr (file:///C:/Users/jack/AppData/Local/Temp/2Thv6hlJ1oezPDttjjcV30mSHOV/resources/app.asar/dist/assets/index.b4e35685.js:1146:158972)
```

### What areas of FiftyOne does this bug affect?

-   [ ] `App`: FiftyOne application issue
-   [ ] `Core`: Core Python library issue
-   [ ] `Server`: FiftyOne server issue

### Willingness to contribute

The FiftyOne Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the FiftyOne codebase?

-   [ ] Yes. I can contribute a fix for this bug independently
-   [ ] Yes. I would be willing to contribute a fix for this bug with guidance from the FiftyOne community
-   [x] No. I cannot contribute a bug fix at this time

**Comment by benjaminpkane** [comment]
Hi @pigOnTuyere. Clearing your browser cache and using the latest `fiftyone-desktop` will likely solve your issues

**Comment by benjaminpkane** [comment]
Resolved by #3683
